sdk/python/kfp/cli/run.py

```diff
@@ -127,6 +131,15 @@ def _display_run(client, namespace, run_id, watch, output_format):
         _print_runs([run], output_format)
 
 
+def _wait_for_run_completion(client, run_id, timeout, output_format):
+    """
+    wait until run status became one of ['succeeded', 'failed', 'skipped', 'error'],
+    or specified timeout reached
+    """
```

Comment:
nit: Given this is only a private method and a very thin wrapper (only 2 lines) over existing methods. I'd rather remove the docstrings.

Reply:
I agree :) DONE